src/webextension/widgets/toolbar.css

```diff
@@ -10,7 +10,7 @@
 }
 
 .toolbar > * + * {
-  margin-inline-start: 5ch;
+  margin-inline-start: 20px;
```

Comment:
With this change, could you also update the padding on the `.toolbar` to be `10px`? That way, we're using consistent units.